StudyBuddy/app/src/main/java/uw/studybuddy/MainActivity.java

```diff
@@ -41,7 +56,13 @@
         NavigationView.OnNavigationItemSelectedListener {
 
     // Data required for the app
-    private UserInfo user;
+    private UserInfo user = new UserInfo();
+
+    DatabaseReference mUserRootRef = FirebaseDatabase.getInstance().getReference().child("Users")
```

Comment:
I meant that this code `FirebaseDatabase.getInstance().getReference().child("Users")` should be in `FirebaseInstance` class

We don't want any database-related code like this part `.child("Users")`in other places.